src/multilspy/lsp_protocol_handler/lsp_types.py

```diff
@@ -31,7 +31,7 @@
 
 from enum import Enum, IntEnum, IntFlag
 from typing import Dict, List, Literal, Union
-from typing_extensions import NotRequired, TypedDict
+from typing import NotRequired, TypedDict
```

Comment:
This is not right!

https://peps.python.org/pep-0655/ only introduced in python 3.11.